### PR TITLE
Update to 2.6.0

### PR DESCRIPTION
**Pull request by Jrice1317**
cookiecutter 2.6.0

**Destination channel:** {defaults}

### Links

- [PKG-4281](https://anaconda.atlassian.net/browse/PKG-4281) 
- [Upstream repository](https://github.com/cookiecutter/cookiecutter/tree/2.6.0)
- [Upstream changelog/diff](https://cookiecutter.readthedocs.io/en/stable/HISTORY.html#id1)

[PKG-4281]: https://anaconda.atlassian.net/browse/PKG-4281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  sha256: db21f8169ea4f4fdc2408d48ca44859349de2647fbe494a9d6c3edfc0542c21c
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
   entry_points:
-    - cookiecutter = cookiecutter.cli:main
+    - cookiecutter = cookiecutter.__main__:main
+  skip: True  # [py<37]
```
Comment (SC426) [inline recipe/meta.yaml:17]: This is ok but not super necessary since we don't build anything less than 3.8 (which we'll also probably stop doing soon).

Comment (SC426) [reply recipe/meta.yaml:17]: I suppose it's good to include what upstream does though.

**Review by SC426 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml:17]: Thanks! I was just about to say that was the only reason I included it. And I know it's not a strict rule, but the consensus among the team seems to be to match upstream.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:17]: You are right @Jrice1317. I think It's preferable to stick to what upstream says.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e61e9034748e3f41b8bd2c11f00d030784b48711c4d5c42363c50989a65331ec
+  sha256: db21f8169ea4f4fdc2408d48ca44859349de2647fbe494a9d6c3edfc0542c21c
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
```
[the remark this answers is not in the document]
Comment (Jrice1317) [reply recipe/meta.yaml:14]: I saw [this](https://github.com/cookiecutter/cookiecutter/blob/2.6.0/setup.cfg#L13) and figured it wouldn't hurt.

**Comment by Jrice1317** [comment]
Thanks for the reviews, guys!

File: recipe/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:14]: I hurts yes. `-vvv` can be very verbose, which can slow down a build and generate unnecessary logs which can slow down databases or cost more money.

Also, the `-vvv` you point at is for pytest, to pip.

**Pull request merged**